manager/dispatcher/dispatcher.go

```diff
@@ -237,27 +234,19 @@ func (d *Dispatcher) Stop() error {
 	d.cancel()
 	d.mu.Unlock()
 	d.nodes.Clean()
-	// wait for all handlers to finish their raft deals, because manager will
-	// set raftNode to nil
-	d.wg.Wait()
 	return nil
 }
 
-func (d *Dispatcher) addTask() error {
+func (d *Dispatcher) canAddTask() error {
```

Comment:
I think this func has somewhat wrong name now. It should be like isRunningLocked. It's just a guard to not accept new requests after stop is called, other requests will be gracefully cancelled by raft.